config/crd/bases/druid.gardener.cloud_etcds.yaml

```diff
@@ -3,15 +3,23 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: etcds.druid.sapcloud.io
+  name: etcds.druid.gardener.cloud
 spec:
-  group: druid.sapcloud.io
+  preserveUnknownFields: false
+  group: druid.gardener.cloud
   names:
     kind: Etcd
     plural: etcds
   scope: ""
   subresources:
     status: {}
+    scale:
+      # specReplicasPath defines the JSONPath inside of a custom resource that corresponds to Scale.Spec.Replicas.
+      specReplicasPath: .spec.replicas
+      # statusReplicasPath defines the JSONPath inside of a custom resource that corresponds to Scale.Status.Replicas.
+      statusReplicasPath: .status.replicas
+      # labelSelectorPath defines the JSONPath inside of a custom resource that corresponds to Scale.Status.Selector.
+      # labelSelectorPath: .status.labelSelector
```

Comment:
Why is this commented? This should be required, right?

Reply:
Nope. its optional.

Reply:
As Amshu suggested with uncomment it.

Reply:
👍 Thanks.